linux/LinuxMDEparser/README.MD

```diff
@@ -0,0 +1,51 @@
+# LinuxMDEparser
+Microsoft Defender Endpoint Log Parser for Linux.
+
+# About
+The LinuxMDEparser is a Windows tool that allows you to export or parse MDE for Linux log files to Excel, making it easier to filter and search the logs. After collecting the MDE logs from any Linux machine, you can then run this tool on your on Windows.
+
+# Usage
+LinuxMDEparser {Command} <option>
+
+EX: LinuxMDEparser.exe real-time-protection --convert
```

Comment:
.exe is for Windows only, please remove to make it compatible with all supported platforms

Reply:
Moved the .exe references to the README.MD inside the Dist forlder